SamplifyAPIClient/Query.cs

```diff
@@ -82,14 +82,16 @@ public QueryOptions(uint offset, uint limit)
             this.Limit = limit;
         }
 
-        public void AddFilter(string field, object value)
+        public QueryOptions AddFilter(string field, object value)
         {
             this.FilterBy.Add(new Filter(field, value));
+            return this;
         }
 
-        public void AddSort(string field, SortDirection direction)
+        public QueryOptions AddSort(string field, SortDirection direction)
```

Comment:
why is this return type changed?

Reply:
We can chain for queries. Good design pattern.